roles/edpm_extra_mounts/meta/argument_specs.yml

```diff
@@ -0,0 +1,33 @@
+---
+argument_specs:
+  # ./roles/edpm_extra_mounts/tasks/main.yml entry point
+  main:
+    short_description: The main entry point for the edpm_extra_mounts role.
+    description: Multiple lines description
```

Comment:
you can drop this if you don't have a long description for the role,

Reply:
done